### ISSUE TITLE
Python3 incompatible codes in egs/*/local

### ISSUE THREAD

**Issue by kamo-naoyuki** [espnet/espnet#558]
```
./egs/voxforge/asr1/local/filter_text.py:22:13: H233  Python 3.x incompatible use of print operator
./egs/voxforge/asr1/local/make_trans.py:20:5: H233  Python 3.x incompatible use of print operator
./egs/voxforge/asr1/local/make_trans.py:49:5: H233  Python 3.x incompatible use of print operator
./egs/csj/asr1/local/csj_rm_tag.py:25:9: H233  Python 3.x incompatible use of print operator
./egs/csj/asr1/local/csj_rm_tag.py:26:9: H233  Python 3.x incompatible use of print operator
 ```

are still incompatible with python3. 

I can change all of them 2/3 compatible, but this is related to the development policy.

How do we write any policy in Readme.md or plan to throw away python2, as [numpy](https://docs.scipy.org/doc/numpy-1.14.1/neps/dropping-python2.7-proposal.html) or [pandas](https://pandas.pydata.org/pandas-docs/stable/install.html)?

(In my opinion)
we have to support python3, at least, in all python scripts because python2 will be deprecated language in 2020, now is 2019, and I even think now is already the time to throw away python2 for espnet. It takes too large cost to make compatible with python2/3 and we can't use any features of python3 forever if keeping to support python2.

**Comment by sw005320** [comment]
Thanks for the suggestion.
I'm still hesitating because these are simply copied from Kaldi. If there are some update in the Kaldi, we may have some conflicts due to our own changes. Also, we may need more work if we also try to maintain these local scripts, since local scripts are highly depending on their tasks and corpora, and these coding styles are very different. But maybe i'm wrong
Sorry for the negative comments.

@kan-bayashi, @ShigekiKarita, I appreciate your inputs.

**Comment by sw005320** [comment]
I agree with this idea of gradually changing it.
1st season change is a minimal change, but greatly contributes to avoid confusions.
Could you work on it then?

**Comment by kamo-naoyuki** [comment]
O.K, I'll work for "1st season" anyway. As far 2nd, I'll discussed at that time, perhaps it is not necessary for local/*.py. 

(Is "1st season" strange in English?  1st step?)

So, should I change the shebang of all local/*.py?
I think some scripts are python2/3 compatible with confidence, for example,` ljspeech/local/**.py` may be compatible, but I don't know which one is compatible or not.

**Comment by kamo-naoyuki** [comment]
I noticed that Kaldi's local  already became python2/3 compatible.

https://github.com/kaldi-asr/kaldi/pull/2925
https://github.com/kaldi-asr/kaldi/blob/master/egs/voxforge/s5/local/make_trans.py

**Comment by sw005320** [comment]
I see. Then we can just copy it.

**Comment by kamo-naoyuki** [comment]
Solved at #567
